DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Delegates/DelegateProgress/EditDelegateCourseAdminFieldViewModel.cs

```diff
@@ -0,0 +1,75 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.DelegateProgress
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using DigitalLearningSolutions.Data.Models.Courses;
+    using DigitalLearningSolutions.Web.Models.Enums;
+    using DigitalLearningSolutions.Web.ViewModels.Common.ViewComponents;
+
+    public class EditDelegateCourseAdminFieldViewModel : EditDelegateCourseAdminFieldFormData
+    {
+        public EditDelegateCourseAdminFieldViewModel(
+            int progressId,
+            int promptNumber,
+            DelegateCourseDetails details,
+            DelegateAccessRoute accessedVia,
+            int? returnPage
+        ) : base(details, returnPage)
+        {
+            var courseAdminField = details.CourseAdminFields.Find(c => c.PromptNumber == promptNumber);
+
+            Options = courseAdminField!.Options;
+            Answer = GetAnswer(details.DelegateCourseInfo, promptNumber);
+            Radios = GetRadioItems(details.DelegateCourseInfo, promptNumber);
+            PromptText = courseAdminField.PromptText;
+            ProgressId = progressId;
+            AccessedVia = accessedVia;
+            ReturnPage = returnPage;

```

Comment:
e.g. don't set this here since it's being passed to the base constructor

Reply:
removed this line and passed returnPage to the base formData